openlogin/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { BrowserContext } from "@playwright/test";
+
+async function confirmEmail({
+  context,
+  timestamp,
+  resend,
+}: {
+  context: BrowserContext;
+  timestamp: number;
+  resend: () => Promise<void>;
+}) {
```

Comment:
Please move this to `/index.utils.ts` - which contains all external test utility atm.